fractal-theme/assets/js/src/search.js

```diff
@@ -0,0 +1,109 @@
+import Fuse from 'fuse.js'
+
+function fractalSearch() {
+  const searchElement = document.querySelector('.search');
+  const searchInput = searchElement.querySelector('#searchInput');
+  const searchSuggestions = searchElement.querySelector('#searchSuggestions');
+  const searchExplanation = document.querySelector('.search-color-meaning');
+  const searchExplanationClose = searchExplanation.querySelector('.search-color-meaning__close');
+  const searchForm = document.querySelector('#searchForm');
+  const SearchApi = '/assets/theme/js/search.json';
+
+  // Search options
+  const searchOptions = {
+    keys: [{
+      name: 'data.title',
+      weight: 0.5
+    }, {
+      name: 'data.primaryKeywords',
+      weight: 0.4
+    }, {
+      name: 'data.secondaryKeywords',
+      weight: 0.1
+    }]
+  };
+
+  // Fetch endpoint
+  fetch(SearchApi)
+    .then(response => {
+      return response.json()
+    })
+    // Search init
+    .then(search => {
+      const fuse = new Fuse(search, searchOptions)
+
+      searchInput.addEventListener('input', (e) => {
```

Comment:
Possible event bug: The same event listener seams to be added more that ones.

Reply:
It's two different inputs on the same page. One in navigation and one on the results page. This is on purpose :)